core/src/structures/transaction_sent_info.rs

```diff
@@ -7,7 +9,7 @@ pub type WireTransaction = Vec<u8>;
 pub struct SentTransactionInfo {
```

Comment:
why is it called **Sent**? this reads to me like it happened already but the actually that struct contains the transaction to be sent